osbs/utils.py

```diff
@@ -187,30 +190,44 @@ def buildconfig_update(orig, new, remove_nonexistent_keys=False):
                 orig[k] = v
 
 
+def check_output_with_retry(args, max_retries, backoff_factor):
+    for counter in range(max_retries + 1):
+        try:
+            subprocess.check_output(args)
+            break
+        except subprocess.CalledProcessError as ex:
+            if counter != max_retries:
+                logger.info("retrying command '%s':\n '%s'", args, ex.output)
```

Comment:
would be nice here to add info `retrying command in X seconds` if sleep time is 0<.